src/app/fdctl/config.h

```diff
@@ -70,9 +70,9 @@ typedef struct {
     int    snapshot_fetch;
     int    genesis_fetch;
     int    poh_speed_test;
-    char   expected_genesis_hash[ 32 ];
+    char   expected_genesis_hash[ 64 ];
```

Comment:
Are these base58 encoded or binary? If binary, they should be `uchar`. If base58, the array size should be `FD_BASE58_ENCODED_32_SZ` I think